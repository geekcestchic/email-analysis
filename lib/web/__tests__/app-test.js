import React from 'react'
import App from '../components/app'
import renderer from 'react-test-renderer'
import axios from 'axios'
const component = renderer.create(<App/>)

it('renders correctly', () => {
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

it('initially says Loading when component mounts', () => {
  let tree = component.toJSON()
  var supposedlyTheLoadingComponent = tree.children[0].children[0]
  expect(supposedlyTheLoadingComponent).toMatch('Loading Orders...')
})

it('makes a call to the API fine', () => {
  axios.get('/orders')
  .then((orders) => {
    expect(orders.data).toBeTruthy()
  })
})
