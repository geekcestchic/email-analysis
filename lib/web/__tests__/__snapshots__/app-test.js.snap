exports[`test renders correctly 1`] = `
<div
  className="content-wrapper">
  <div>
    Loading Orders...
  </div>
</div>
`;
